.travis.yml

```diff
@@ -11,9 +11,21 @@ cache:
 
 install: true
 
-script:
-  - travis_wait 10 mvn clean install -DskipTests=true
-  - travis_wait 10 mvn test
+# define two parallel jobs
+matrix:
+  include:
+    # This job is to install and test project
+    - os: linux
+      env: CUSTOM_JDK="oraclejdk8"
+      script:
+        - mvn clean install
```

Comment:
skip test please

Reply:
have removed  `mvn test`

Reply:
should we use `CUSTOM_JDK=openjdk8` in `env`?

Reply:
Why?

Reply:
Problem solved.